app/controllers/pages_controller.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 class PagesController < ApplicationController
-  def home
```

Comment:
This will break the home page, won't it? The root route is to home

Reply:
This did not break the homepage.

Reply:
Although I have put it back.